tests/ci_tests/test_coffee_maker_mains.py

```diff
@@ -0,0 +1,45 @@
+"""Tests ensuring each coffee_maker script with a __main__ guard imports cleanly."""
+
+from __future__ import annotations
+
+import os
+import runpy
+from pathlib import Path
+
+import pytest
+
+_REPO_ROOT = Path(__file__).resolve().parents[1].parent
+_COFFEE_MAKER_DIR = _REPO_ROOT / "coffee_maker"
+
+
```

Comment:
Added docstring for the function 'count_words'.
```suggestion
def count_words(input_string: str) -> int:
```